src/dbnode/persist/fs/commitlog/read_write_prop_test.go

```diff
@@ -51,6 +52,9 @@ type seriesWritesAndReadPosition struct {
 }
 
 func TestCommitLogReadWrite(t *testing.T) {
+	// Make sure we're not leaking goroutines
+	defer leaktest.CheckTimeout(t, time.Second)()
```

Comment:
This is a common fault, unfortunately you will capture the start of the leaktest at the end of the test.